src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/OasysService.kt

```diff
@@ -154,22 +157,10 @@ class OasysService(
   }
 
   fun getAssessmentId(prisonNumber: String): Long? {
-    val assessments = when (val result = oasysApiClient.getAssessments(prisonNumber)) {
-      is ClientResult.Failure -> {
-        log.warn("Failure to retrieve Assessment for $prisonNumber reason ${result.toException().cause}")
-        throw NotFoundException("No assessment found for prison number: $prisonNumber")
-      }
-
-      is ClientResult.Success -> AuthorisableActionResult.Success(result.body)
-    }
+    val oasysAssessment = getAssessments(prisonNumber)
```

Comment:
can we just tweak the name of oasysAssessment  - either by making it plural or making it assessmentTimeline. It's just misleading having it as assessment when it is a collection of assessments